apollo-gradle-plugin/src/test/kotlin/com/apollographql/apollo3/gradle/test/KotlinPluginVersionTests.kt

```diff
@@ -17,12 +17,7 @@ class KotlinPluginVersionTests {
    */
   @Test
   fun `kotlin n-1 succeeds`() {
-    val currentKotlinVersion = System.getenv("COM_APOLLOGRAPHQL_VERSION_KOTLIN_PLUGIN") ?: "1.6"
-    val kotlinPluginVersionToTest = if (currentKotlinVersion.startsWith("1.7")) {
-      "1.6.0"
-    } else {
-      "1.5.31"
-    }
+    val kotlinPluginVersionToTest = "1.6.0"
```

Comment:
Should this test be `kotlin JVM min version succeeds` and use 1.5? Since this is what we claim 1.5+ support?

Reply:
Good point! ✅